### PR TITLE
[ConfigBundle] Fixed variable name, in routing and view, for ConfigController::indexAction.

### PR DESCRIPTION
**Pull request by adrianwaler**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #1418

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -94,7 +94,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "4.0-dev"
+            "dev-master": "4.1-dev"
```
Comment (sandergo90) [inline composer.json]: Don't commit this please

**Comment by adrianwaler** [comment]
@sandergo90 Internal name was changed here https://github.com/Kunstmaan/KunstmaanBundlesCMS/commit/0cd02390dacbf1ae997f7d351d7bc3bc0eea38696 and that's why it now don't work properly. More about you can read in #1418

**Comment by sandergo90** [comment]
Hi @adrianwaler Yes I know ;) My comment is on the part of dev-master that you are trying to commit. And "Don't forget to change the internal_name in the ConfigController".

Cheers

**Comment by sandergo90** [comment]
@adrianwaler last think, can you update your commit message to "ConfigBundle" instead of "ConfigBunlde" ? ;)

Cheers

**Comment by adrianwaler** [comment]
@sandergo90 I got it ^^ I don't know why I didn't find it earlier. 

Yeah sure, I'll change it. Thanks!

**Pull request merged**